docs/docs/configuration/systask.md

```diff
@@ -69,6 +69,80 @@ The task takes 3 parameters:
 }
 ```
 
+## Switch
+A switch task is similar to `Decision` task with the exception that it uses the extensible expression evaluation 
```

Comment:
SWITCH task would replace DECISION task which would be marked as deprecated. Please replace the section for DECISION with SWITCH in the documentation.

Reply:
Done.